static/index.html

```diff
@@ -4,6 +4,7 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
   <link rel="stylesheet" href="stylesheets/style.css">
   <link href="https://fonts.googleapis.com/css?family=Muli:200,400|Saira:400,700" rel="stylesheet">
+  <link rel="stylesheet" href="lib/font-awesome/css/font-awesome.min.css">
```

Comment:
Let's use the cdn version of font awesome. so we don't need to store it, and users have a better performance.

Reply:
Done